src/appshell/internal/notationpagestate.cpp

```diff
@@ -46,7 +46,13 @@ bool NotationPageState::isPanelVisible(PanelType type) const
         return configuration()->isNotationNavigatorVisible().val;
     case PanelType::NotationStatusBar:
         return configuration()->isNotationStatusBarVisible().val;
-    default:
+    case PanelType::Mixer:
+    case PanelType::TimeLine:
+    case PanelType::Synthesizer:
+    case PanelType::SelectionFilter:
+    case PanelType::Piano:
+    case PanelType::ComparisonTool:
+    case PanelType::Undefined:
```

Comment:
with that you could add the `return false` outside the switch, so the code still works as intended if more types get added, even if the warning that there is an unhandled case got ignored or overseen

Reply:
`return false` is already outside the switch

Reply:
Ah, right, you removed it in an eralier commit, but reinstated it in this. Sorry for the noise...